opensrp-brac-hnpp/src/main/java/org/smartregister/unicef/dghs/service/VaccineDueUpdateIntentService.java

```diff
@@ -34,38 +34,38 @@ protected void onHandleIntent(Intent intent) {
        //need to get all due vaccine name,date from alert table group by baseentityid
         String query = "select base_entity_id,dob from ec_child";
         ArrayList<String[]> childs = new ArrayList<>();
-        Cursor cursor = null;
-        try{
-            cursor = HnppApplication.getInstance().getRepository().getReadableDatabase().rawQuery(query, new String[]{});
-            if(cursor !=null && cursor.getCount() >0){
+        try (Cursor cursor = HnppApplication.getInstance().getRepository().getReadableDatabase().rawQuery(query, new String[]{})) {
+            if (cursor != null && cursor.getCount() > 0) {
                 cursor.moveToFirst();
                 String[] member = new String[2];
                 while (!cursor.isAfterLast()) {
                     member[0] = cursor.getString(0);
                     member[1] = cursor.getString(1);
+
+                    if (!TextUtils.isEmpty(member[1])) {
+                        DateTime dateTime = new DateTime(member[1]);
+                        VaccineSchedule.updateOfflineAlerts(member[0], dateTime, "child");
+                        //ServiceSchedule.updateOfflineAlerts(member[0], dateTime);
+                    }
+
```

Comment:
remove this from here. as one cursor already open don't use cursor inside other cursor